source/test/TestApp/TestUtils.cpp

```diff
@@ -3,8 +3,45 @@
 #include "TestAppPch.h"
 #include "TestUtils.h"
 
+#include <random>
```

Comment:
Seems like this is also using a pch.  Any reason not to put these includes into the pch?